staging/gatekeeper/README.md

```diff
@@ -5,7 +5,7 @@ Kubernetes.
 
 ## Prerequisites
 
-- Kubernetes 1.15 (or newer) for validating and mutating webhook admission
+- Kubernetes 1.9 (or newer) for validating and mutating webhook admission
```

Comment:
1.9 ? do we support that version in any of our products 🤔 ?

Reply:
This was a request from helm upstream to support it because it does work with that.